### ISSUE TITLE
#474. Ones and Zeroes

### ISSUE THREAD

**Issue by stuti-v4** [tanyagupta0201/LeetCode-Problems-Solutions#480]
You are given an array of binary strings strs and two integers m and n.

Return the size of the largest subset of strs such that there are at most m 0's and n 1's in the subset.

A set x is a subset of a set y if all elements of x are also elements of y.

**Comment by utkarshaskumbhar** [comment]
Assign me this issue

**Comment by tanyagupta0201** [comment]
> Assign me this issue

fcfs

**Comment by utkarshaskumbhar** [comment]
> > Assign me this issue
> 
> fcfs

Do I have to create my own issue in this repository ??

**Comment by tanyagupta0201** [comment]
> > > Assign me this issue
> > 
> > 
> > fcfs
> 
> Do I have to create my own issue in this repository ??

Yes, you may create an issue for the questions that are not already present or assigned already.
